libcontainer/container_linux.go

```diff
@@ -228,6 +228,14 @@ func (c *linuxContainer) Set(config configs.Config) error {
 }
 
 func (c *linuxContainer) Start(process *Process) error {
+	// configure linux personality before starting the process
+	if c.Config().Personality != nil {
```

Comment:
Why are you using `c.Config().` rather than `c.config.`?

Reply:
@kolyshkin this is my bad. I'll fix this.

Reply:
@kolyshkin resolved in latest commit.